course/models.py

```diff
@@ -935,6 +936,11 @@ def enroll_from_sis(self) -> Tuple[int, int]:
         delcount = 0
         try:
             participants = sis.get_participants(self.sis_id)
+        except HTTPError as exc:
+            code = exc.response.status_code
+            # pylint: disable-next=logging-fstring-interpolation
+            logger.exception(f"{self}: Error {code} when getting participants from SIS.")
```

Comment:
Why not do this in the linter-preferred way using %s and %d in the string, and then avoid linter exception? Also, since we are editing this function in any case, also fixing the below similar old log message.